src/StatusQ/Popups/StatusPopupMenu.qml

```diff
@@ -40,6 +41,16 @@ Menu {
         statusPopupMenu: root
     }
 
+    contentItem: ListView {
```

Comment:
StatusListView pls, so that we get a consistent behavior (and styled scrollbars etc)

Reply:
This ListView is part of StatusPopupMenu - I actually added this new contentItem just to have correct logic for `interactive` property. As I know we dont have design for scrollable account menu - or other menus. Are you sure I need to reuse StatusListView here ?

Reply:
But you also added the scrollbars :)

Yeah, just use `StatusListView` and adjust the `interactive` property